setup.py

```diff
@@ -83,7 +83,7 @@ def absolute_links(txt):
             "pysb",
         ],
         "quality": [
-            "flake8>=3.8.3",
+            "pre-commit",
```

Comment:
Is the `ruff` dependency somehow managed by `pre-commit` and the ruff-pre-commit repo?

https://github.com/astral-sh/ruff-pre-commit/blob/e76e45000a5aa92cd2ec4d3fb1485f2f9fea5097/pyproject.toml#L1-L6

Reply:
correct